controllers/apply/under10k/fields/your-idea.js

```diff
@@ -89,174 +89,61 @@ module.exports = {
             ],
         });
     },
-    fieldYourIdeaPriorities(locale, data = {}, flags = {}) {
+    fieldYourIdeaPriorities(locale) {
         const localise = get(locale);
 
-        const projectCountry = get('projectCountry')(data);
-        const supportingCOVID19 = get('supportingCOVID19')(data);
+        // not needed currently const projectCountry = get('projectCountry')(data);
```

Comment:
We can probably just remove this line, no need to comment things out like this when we can just revisit the history on Git etc!